Userland/Applications/DisplaySettings/EffectsSettingsWidget.cpp

```diff
@@ -130,14 +130,14 @@ ErrorOr<void> EffectsSettingsWidget::load_settings()
     for (size_t i = 0; i < list.size(); ++i)
         TRY(m_geometry_list.try_append(TRY(String::from_utf8(list[i]))));
     m_geometry_combobox->set_model(ItemListModel<String>::create(m_geometry_list));
-    m_geometry_combobox->set_selected_index(m_system_effects.geometry());
+    m_geometry_combobox->set_selected_index(static_cast<size_t>(m_system_effects.geometry()));
```

Comment:
All of the static_cast's in this commit and following commits can use `to_underlying` for this purpose, so we don't need to keep repeating the casted-to type and don't need to keep it in sync with the enum's underlying type if it ever changes:
```c++
m_geometry_combobox->set_selected_index(to_underlying(m_system_effects.geometry()));
```